Applications/ReinforcementLearning/DeepQ/jni/main.cpp

```diff
@@ -499,8 +503,9 @@ int main(int argc, char **argv) {
         nntrainer::Tensor in_tensor;
         try {
           in_tensor = nntrainer::Tensor(inbatch);
-          mainNet.forwarding({MAKE_SHARED_TENSOR(in_tensor)}, {Q});
-          mainNet.backwarding(iter);
+          ml::train::RunStats stat;
+          mainNet.forwarding(stat, {MAKE_SHARED_TENSOR(in_tensor)}, {Q});
```

Comment:
we do not need stat variables everywhere. plz move it to the end and set nullptr as an default.